src/helpers/request-spy.js

```diff
@@ -0,0 +1,51 @@
+import assign from 'lodash.assign';
+import sinon from 'sinon';
+
+
+const limitOffsetResponse = require('../fixtures/limit-offset-response.json');
```

Comment:
The current loader spec only supports importing javascript.

Weather or not this will work:
```
import limitOffsetResponse from '../fixtures/limit-offset-response.json';
```

Will depend on your toolchain:
* :+1: babel > ES2015 > browserify (because babel compiles it down to `require`)
* :x: SystemJS will fail unless you use a json plugin

I just opened a ticket annoying the spec writers, asking if JSON will ever get special treatment:
https://github.com/whatwg/loader/issues/133

With that said, I'm in favor of using an `import` declaration and load the json as a "data module" at the top alongside all other module imports.